google-marketplace/test-marketplace.sh

```diff
@@ -10,6 +10,6 @@ fi
 REGISTRY=gcr.io/datadog-sandbox/datadog
 FULL_TAG=$1
 
-gcrane cp datadog/operator:$FULL_TAG $REGISTRY/datadog-operator:$FULL_TAG
-docker build --pull --no-cache --build-arg TAG="$FULL_TAG" --tag $REGISTRY/deployer:$FULL_TAG . && docker push $REGISTRY/deployer:$FULL_TAG
-mpdev verify --deployer=$REGISTRY/deployer:$FULL_TAG --parameters='{"name": "datadog", "namespace": "datadog-agent", "datadog.credentials.apiKey": "00000000000000000000000000000000"}'
+gcrane cp "datadog/operator:$FULL_TAG" "$REGISTRY/datadog-operator:$FULL_TAG"
+docker build --pull --no-cache --build-arg TAG="$FULL_TAG" --tag $REGISTRY/deployer:"$FULL_TAG" . && docker push $REGISTRY/deployer:"$FULL_TAG"
```

Comment:
I think shellcheck doesn’t complain about `$REGISTRY` not being quoted because it is set just above and shellcheck can check that its value is space free.
So, this suggestion is mostly for style consistency.

```suggestion
docker build --pull --no-cache --build-arg TAG="$FULL_TAG" --tag "$REGISTRY/deployer:$FULL_TAG" . && docker push "$REGISTRY/deployer:$FULL_TAG"
```